test/db/cmd/cmd_afl

```diff
@@ -158,7 +158,7 @@ FILE=bins/elf/ls
 CMDS=<<EOF
 e analysis.jmp.cref=true
 aaa
-aflj~{main}
+aflj~{[0]}
```

Comment:
At https://github.com/rizinorg/rizin/pull/1118/files#diff-e5fc91db395b03c4d8f7cd9482f62f84a4797fd2d9859973b0960d442a171ba3R510 it says that to simplify the grep syntax we can omit `[..]`. Is this the case? Do we have some tests for that syntax?